core/src/main/java/io/onetable/hudi/HudiTableManager.java

```diff
@@ -96,9 +97,12 @@ HoodieTableMetaClient initializeHudiTable(String tableDataPath, OneTable table)
     keyGeneratorClass =
         getKeyGeneratorClass(
             table.getPartitioningFields(), table.getReadSchema().getRecordKeyFields());
+    boolean hiveStylePartitioningEnabled =
```

Comment:
Can you update the tests to validate this?

Reply:
yeah did that.